draft-ietf-rats-architecture.md

```diff
@@ -1286,14 +1286,19 @@ For a discussion on the security of epoch IDs see {{epochids-sec}}.
 The conveyance of Evidence and the resulting Attestation Results
 reveal a great deal of information about the internal state of a
 device as well as potentially any users of the device.
+
 In many cases, the whole point of attestation procedures is
 to provide reliable information about the type of the device and the
 firmware/software that the device is running.
+
 This information might be particularly interesting to many attackers.
 For example, knowing that a device is
 running a weak version of firmware provides a way to aim attacks better.
 
-Many Claims in Evidence, many Claims in Attestation Results, and Appraisal Policies potentially contain
+In some circumstances, if an attacker can become aware of appraisal policies, it could potentially provide an attacker with insight into defensive mitigations.
+It is recommended that attention be paid to confidentiality of policies.
+
+Additionally, many Claims in Evidence, many Claims in Attestation Results, and Appraisal Policies potentially contain
```

Comment:
The same (both line 1298 and line 1301) can be said of Endorsements and Reference Values.   That's why you modified line 1423 but the same applies here too I think.

Reply:
I pushed another commit to update this. I meant to do it as a Suggestion, but github didn't let me.
(I eventually realized that I had gotten signed out!)
Please review again. Oxford comma included.